packages/gatsby-plugin-graphql-config/index.js

```diff
@@ -0,0 +1,2 @@
+// no-op
+"use strict";
```

Comment:
I think we don't need `index.js` here as long as we have `src/index.ts`. It will be compiled to `index.js` on build.

Reply:
the `.gitignore` in the other typescript plugin i found (page builder) did not include this for some reason? so it's checked in, despite being transpiled by babel